src/common/parse-util.c

```diff
@@ -0,0 +1,36 @@
+#include "parse-util.h"
+#include <limits.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int parse_long(const char *in, long *val) {
+        if (in == NULL || strlen(in) == 0) {
+                return false;
+        }
+
+        const int decimal_base = 10;
+        char *ptr = NULL;
+        *val = strtol(in, &ptr, decimal_base);
+        if (strcmp(ptr, "") != 0) {
```

Comment:
From the system coding guidelines:
```
Please use streq() and strneq() instead of strcmp(), strncmp() where applicable (i.e. wherever you just care about equality/inequality, not about the sorting order).
```

Reply:
I have removed this strcmp() usage, but still added string-util.h with streq() and strneq() macros - for future use, and also to introduce string-util.h - so that anyone who wants to create nice string util wrapper - can place it there